run_server.py

```diff
@@ -0,0 +1,2 @@
+from RuddockWebsite import app
+app.run(debug=True, port=4999)
```

Comment:
I explained this in the commit message 90fd389016306817e1c522b285c3b576baddb5e7 -- this is only run when testing locally (wsgi does not run it), so we can commit debugging things and make `run_server.py` however best for testing.